unreleased_history/performance_improvements/fix_11406_perf_regression.md

```diff
@@ -0,0 +1 @@
+Fixed higher read QPS during DB::Open() reading files created prior to #11406, especially when reading many small file (size < 52 MB) during DB::Open() and partitioned filter or index is used.
```

Comment:
On the main branch, this would be a release note for the next minor release. But this will be fixed in a patch release on the current minor release, so from the minor release over minor release perspective, I consider it redundant for most users, because they will be upgrading from a patched version.

Reply:
Oh right - good point - will remove this history entry and follow the new procedure to update history for the patch only.

Reply:
Fixed